cherry_picker/cherry_picker/cherry_picker.py

```diff
@@ -16,19 +16,22 @@
 
 from . import __version__
 
-CREATE_PR_URL_TEMPLATE = ("https://api.github.com/repos/"
-                          "{config[team]}/{config[repo]}/pulls")
-DEFAULT_CONFIG = collections.ChainMap({
-    'team': 'python',
-    'repo': 'cpython',
-    'check_sha': '7f777ed95a19224294949e1b4ce56bbffcb1fe9f',
-    'fix_commit_msg': True,
-    'default_branch': 'master',
-})
+CREATE_PR_URL_TEMPLATE = (
+    "https://api.github.com/repos/" "{config[team]}/{config[repo]}/pulls"
```

Comment:
I believe, one-line style doesn't require two chunks to be separated
```suggestion
    "https://api.github.com/repos/{config[team]}/{config[repo]}/pulls"
```